.cirrus.yml

```diff
@@ -111,6 +111,8 @@ task:
     memory: 8G
 
   setup_script: |
+    # EPEL is needed for python2-future, python2-junit_xml, python-flake8 and libbsd-devel.
+    rpm -Uvh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
```

Comment:
Doing `yum -y install <URL>` is probably better than directly calling `rpm`. Feels more robust.

Reply:
Thanks Adrian! I have updated the pull request.